misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormatterUtils.java

```diff
@@ -134,6 +167,14 @@ static DiagnosticPos getPosition(Node node) {
                 return parent;
             }
         }
+        if (parentKind == (SyntaxKind.OBJECT_CONSTRUCTOR) && grandParent != null &&
+                grandParent.kind() == (SyntaxKind.LOCAL_VAR_DECL)) {
```

Comment:
Additional parenthesis around the syntax kind

Reply:
Fixed